examples/kubernetes/clustertemplates/main.tf

```diff
@@ -0,0 +1,9 @@
+data "vkcs_kubernetes_clustertemplates" "templates" {}
+
+output "available_templates_by_name" {
```

Comment:
could we have the only output with linked name and version?

Reply:
done